### DISCUSSION TITLE
Selwyn in Ultima VII?

### DISCUSSION THREAD

**Discussion by FurnaceFox** [exult/exult#536] [General]
Just curious as to people's thoughts; we will likely never know what was intended...

Does Selwyn the mage actually appear in the game? He apparently leads the conspirators in Despise/Shame who plan on overthrowing Lord British.

A mage is encountered on the lower floor of the conspirators' lair, but he is not named (a "monster" with no NPC number) and is spawned only-once by an egg. Could this be Selwyn?

Up the invisible staircase, four conspirators are found seated at a table. The paladin is named Brax, the pirate is named Fransisco, the fighter is named Corwin, and the mage is named Korryn. One scroll found on the table is Selwyn's last will. The other scroll is an agreement between four individuals: Fransisa (likely a misspelling of Fransisco), Corwin, Brax, and Athelas. Selwyn has not signed it. The questionable names are Corwin (which is actually a pretty close alternative spelling to Korryn) and Athelas (which is the name of the medicinal herb in Tolkien's Middle-earth; also known as kingsfoil). The names Korryn and Selwyn share the same ending (yn), but that's likely a stretch. Athelas apparently does not exist in the game, aside from the name given on the scroll. Obviously some "mistakes" or omissions may have been made by the developers, some of which made it into the final game.

**Comment by FurnaceFox** [comment]
I guess I'd just like to add that I assumed the ghost spawned once-only by the egg in Selwyn's Tower is Selwyn's "pet" (as described in his last will) and not the ghost of Selwyn himself.

Similarly if you take the southern teleporter (not the mirror) in the dining area of the conspirator's hideout, you end up in an area of the dungeon with sleep fields and eight locked chests. A dead mage in an adjoining room has the key to the chests, and the mage's body has an egg next to it that spawns a ghost.

**Reply by GrantWChapman** [reply]
That's a good question. The Ultima codex has him listed as one of the mages in that encounter. My guess would be that the names were changed in a revision but not updated in the NPC data.

**Reply by FurnaceFox** [reply]
Grant, thanks for the response. I went ahead and updated the NPC data in Exult for use in my own copy of the game.

Brax (NPC 288) was set up correctly already, although I had to turn on "Met" under his NPC flag so when you clicked on him, his actual name was given instead of "paladin". This also will identify him by name if added to the Avatar's party.

Had to do the same for Corwin (NPC 289); turned on "Met" for him otherwise he was just identified as "fighter".  There was a Corwin in Ultima Underworld, but I assume they are not the same person. Korryn might have been preferred to distinguish him from the earlier Corwin in UU but considering the NPC's name matched the name on the scroll in U7, I decided to keep the name as it was.

Fransisco (NPC 290) was set up correctly already. (I assumed the developers' spelling was deliberate; they went with the Latin rather than the Spanish/Portuguese "Francisco".)

I renamed the mage Korryn (NPC 292) to "Selwyn".

Now I'd like to modify the names on the scroll... "Fransisa" to "Fransisco", " Brax" to "Brax" (there was an extra space in there), and "Korryn" to "Selwyn". As I understand it, I will need to use Usecode C to do so, but have not been able to do so successfully. I guess it's not compiling correctly, and I can't get the game to load after the changes are made. Fortunately, I had created a backup of the usecode file.

I remember playing the game as a teenager shortly after the game's release and being disappointed that you couldn't bring evidence of the plot to Lord British's attention in the original game. But if I can't change the text of a scroll in the game, I'm afraid I am not ready to tackle conversations just yet.

**Comment by wench** [comment]
NPC names as shown by using the met flag were not visible in the original game. The Met flag for NPC names was a feature added in SI. BG did not have it, But In Exult I added it to BG too. For BG the names are really an internal development thing that comes from the NPC data.

**Reply by FurnaceFox** [reply]
Thanks Alexis, it's been so long since I've played the non-Exult version of Ultima. I couldn't recall what was in the original and what elements can be attributed to Exult. I like the fact that you added Met flags in BG. Makes for more realistic gameplay. For a shopkeeper is just a shopkeeper until you meet, speak with and learn his/her name. I was hoping that maybe the developers had originally planned on giving Fransisco and Korryn speaking roles which was why their Met flags were on and the paladin's and the fighter's (Brax and Corwin, respectively) were turned off. But then again, I've tinkered and messed around with my copies of BG and SI so much over the years that I've lost track of my own changes!

**Reply by KnightCaptainU7** [reply]
As a bit of trivia, there are many global flags in BG for "MET" on a per-NPC basis.

**Comment by agentorangeguy** [comment]
> Just curious as to people's thoughts; we will likely never know what was intended...
> 
> Does Selwyn the mage actually appear in the game? He apparently leads the conspirators in Despise/Shame who plan on overthrowing Lord British.
> 
> A mage is encountered on the lower floor of the conspirators' lair, but he is not named (a "monster" with no NPC number) and is spawned only-once by an egg. Could this be Selwyn?
> 
> Up the invisible staircase, four conspirators are found seated at a table. The paladin is named Brax, the pirate is named Fransisco, the fighter is named Corwin, and the mage is named Korryn. One scroll found on the table is Selwyn's last will. The other scroll is an agreement between four individuals: Fransisa (likely a misspelling of Fransisco), Corwin, Brax, and Athelas. Selwyn has not signed it. The questionable names are Corwin (which is actually a pretty close alternative spelling to Korryn) and Athelas (which is the name of the medicinal herb in Tolkien's Middle-earth; also known as kingsfoil). The names Korryn and Selwyn share the same ending (yn), but that's likely a stretch. Athelas apparently does not exist in the game, aside from the name given on the scroll. Obviously some "mistakes" or omissions may have been made by the developers, some of which made it into the final game.

I was under the impression that Selwyn was one of the conspirators but maybe got killed either by the others for being greedy or that they couldn't trust him.  Most likely he's the dead mage found there.  

FYI, at least in the original, you could find those conspirators on top of LB's castle after about a week's game time.  They were on the roof presumably because if an npc was on a higher z level, going to their next schedule would not change it so I would assume they are supposed to be found on the main level.
